package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@deg-skeletor/orko",
-    "version": "1.0.2",
+    "version": "1.0.3",
```

Comment:
Would this be a non-breaking change/enhancement rather than a bug fix?

Reply:
Ah, good call. Updated to 1.2.0